src/components/base/outline-breadcrumbs/outline-breadcrumbs.css

```diff
@@ -0,0 +1,16 @@
+:host,
+a,
+::slotted(a) {
+  @apply font-body text-brand-primary transition-colors duration-300 mr-4 no-underline;
```

Comment:
@sarahp text-brand-primary seems like it shouldn't be applied to `:host`. So that the `/` isn't the link color.

Reply:
Removed references to `:host` and everything is still styled as expected.